sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromETagAndFeedRangeImpl.java

```diff
@@ -82,19 +82,21 @@ public void populateRequest(RxDocumentServiceRequest request) {
     public void populatePropertyBag() {
         super.populatePropertyBag();
 
-        setProperty(
-            this,
-            Constants.Properties.CHANGE_FEED_START_FROM_TYPE,
-            ChangeFeedStartFromTypes.LEASE);
-
-        setProperty(
-            this,
-            Constants.Properties.CHANGE_FEED_START_FROM_ETAG,
-            this.eTag);
-
-        this.feedRange.setProperties(
-            this,
-            true);
+        synchronized(this) {
```

Comment:
Does base class method call need to be "synchronized"/single threaded as well?

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.